testing/src/main/java/org/oppia/testing/OppiaTestRunner.kt

```diff
@@ -0,0 +1,98 @@
+package org.oppia.testing
+
+import android.app.Application
+import android.content.Context
+import android.os.Bundle
+import androidx.test.runner.AndroidJUnitRunner
+import java.lang.reflect.Field
+
+// TODO(#59): Remove this runner once application classes can be specified per-test suite.
```

Comment:
Just to clarify, this is something we could do once we move to Bazel right?

Reply:
Yep, but we need phase 2 to be underway and maybe mostly done before this is possible since it requires pulling apart the manifests to the point where we can actually have per-test manifests. I suspect the runner will be around for a while in the interim.